docs/resolving_merge_conflicts.md

```diff
@@ -1,26 +1,46 @@
 
-The majority of the time, the conflict is trivial - due to addition of new terms to the same point in the file. Becuase terms are ordered in the file by ID, this happens whenever two edits add terms without any intervening IDs.  Trivial clashes are easy to spot - they involve whole term stanzas + declarations.  No content is deleted (although this   
+The majority of the time, the conflict is trivial - due to addition of new terms to the same point in the file. Because terms are ordered in the file by ID, this happens whenever two edits add terms without any intervening IDs. Trivial clashes are easy to spot - they involve whole term stanzas + declarations.  
 
-Occassionally non-trivial clashes will happen when two pull requests include edits to the same term or even the same axiom.  Ask an editor for help if you don't feel confident resolving these.  
+Occassionally non-trivial clashes will happen when two pull requests include edits to the same term or even the same axiom. Ask an editor for help if you don't feel confident resolving these.  
 
 ### SOP.
 
-1. Reserialise the Master file using ROBOT. This will fix any odd orderings, that may have been introduced during previous merges.
-    ```sh
-    sh ./run.sh ROBOT convert --input cl-edit.owl -f ofn --output cl-edit.owl
-   ```
+1. Reserialise the Master file using the ODK commands normalize_src or normalize_obo_src. These commands require installation of ROBOT. If not already installed, follow [the instructions here](http://robot.obolibrary.org).
+
+2. At the line command (PC) or Terminal (Mac), use the cd (change directory) command to navigate to the repository's src/ontology/ directory.
+For example,
+
+'''
+cd PATH_TO_ONTOLOGY/src/ontology/
+'''
+
+Replace "PATH_TO_ONTOLOGY" with the actual file path to the ontology. If you need to orient yourself, use the '''pwd''' (present working directory) or '''ls''' (list) line commands.
+
+3. If you are resolving a conflict in an .owl file, run:
+
+'''
+make normalize_src
+'''
+
+If you are resolving a conflict in an .obo file, run:
+
+'''
+make normalize_obo_src
+'''
+
+
 4. In GitHub Desktop:
 
-   * Checkout Master and pull to make sure your Master is up to date.
-   * Checkout the branch for the pull request & make sure it is up to date.
-   * Choose Branch.update_from_master: ![image](https://user-images.githubusercontent.com/112839/112127621-89af9f00-8bbd-11eb-8613-f3a2b8166085.png)
+   * Checkout Master and pull to make sure your Master branch is up to date.
+   * Checkout the branch for the pull request and make sure it is up to date.
+   * Choose Branch > Update from master: ![image](https://user-images.githubusercontent.com/112839/112127621-89af9f00-8bbd-11eb-8613-f3a2b8166085.png)
```

Comment:
its ok for now, but when you move it to obook, could you download the picture and place it as an actual image file please